minio/minio
# Initialize configs correctly, move notification config

## Description
Initialize configs correctly, move notification config

## Motivation and Context
This PR also removes deprecated tests, adds checks
to avoid races reproduced on CI/CD.

## How to test this PR?
Test fresh setup initialization of configs, and run `make verify` to fix race tests.

## Types of changes

## Checklist:
